js/classes/entities/foreground/dialog/conversation.js

```diff
@@ -55,7 +55,12 @@ class Conversation {
             } else {
                 //currentBubble is a DecisionBubble. No typewriter effect.
                 currentBubble.removeFromWorld = true;
-                this.currentIndex = currentBubble.choices[currentBubble.selected].next;
+                if (typeof currentBubble.choices[currentBubble.selected].next === 'number') {
```

Comment:
next is usually a number. However, sometimes next is a function (when you shop, for example, because to determine which bubble you go to, you gotta check INVENTORY.runes first). This accounts for both possibilities.